py-polars/polars/dataframe/frame.py

```diff
@@ -5309,6 +5309,9 @@ def join_asof(
           - A "forward" search selects the first row in the right DataFrame whose
             'on' key is greater than or equal to the left's key.
 
+          - A "nearest" search select sth erow in the right DataFrame whose value
```

Comment:
"select sth erow in the" 

(*doesn't seem right)

Reply:
Thanks @ritchie46, this typo was also in lazyframe so I fixed it there as well.